src/gradesStudentsPanel.html

```diff
@@ -0,0 +1,33 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GradeBook</title>
+    <script src="assets/ts/gradesStudentsPanel.ts" async></script>
```

Comment:
You don't have to import script here, webpack does it;)

Reply:
done :)